packages/gov-app/pages/proposals/new.tsx

```diff
@@ -1,16 +1,108 @@
 import type { NextPage } from "next";
-import { Layout } from "@gov-app/libs/components/Layout";
-import { Header } from "@gov-app/libs/components/Header";
+
+import { If } from "react-extras";
+import {
+	Button,
+	Header,
+	Layout,
+	ProposalAdvanced,
+	ProposalDetails,
+	TextField,
+	WalletConnect,
+} from "@gov-app/libs/components";
+import { Spinner } from "@gov-app/libs/assets/vectors";
+import { ChangeEvent, FormEventHandler, useCallback, useState } from "react";
 
 const NewProposal: NextPage = () => {
+	const [proposalTitle, setProposalTitle] = useState<string>("");
+	const [advancedOpen, setAdvancedOpen] = useState<boolean>();
+
+	const { busy, onFormSubmit } = useFormSubmit();
+
 	return (
 		<Layout>
 			<Header />
-			<div className="flex flex-1 items-center justify-center">
-				<h1 className="font-display text-6xl uppercase">New Proposal</h1>
+			<div className="w-full max-w-3xl flex-1 self-center px-8 pb-12">
+				<form onSubmit={onFormSubmit}>
+					<h1 className="font-display mb-8 text-center text-7xl uppercase">
+						Submit a Proposal
+					</h1>
+
+					<p className="mb-8 text-center text-lg">
+						To submit a proposal you must be a CENNZnet Councillor.
```

Comment:
need to extend this a bit, can you append the paragraph with "lorem ipsum" text to make it as place holder for now?